apps/website-25/src/pages/profile.tsx

```diff
@@ -35,6 +44,7 @@ const ProfilePage = withAuth(({ auth }) => {
           <HeroH1>Your profile</HeroH1>
           <HeroH2>See your progress and share your learnings</HeroH2>
         </HeroSection>
+        <Breadcrumbs route={CURRENT_ROUTE} />
```

Comment:
I also added breadcrumbs to the page while I was adding the route:
![Screenshot 2025-04-16 at 10 25 24](https://github.com/user-attachments/assets/0c56e138-b475-432d-b540-c4b489596df5)